lib-multisrc/madara/assets/i18n/messages_es.properties

```diff
@@ -0,0 +1,27 @@
+author_filter_title=Autor
+artist_filter_title=Artista
+year_filter_title=Año de lanzamiento
+status_filter_title=Estado
+status_filter_completed=Completado
+status_filter_ongoing=En curso
+status_filter_canceled=Cancelado
+status_filter_on_hold=En pausa
+order_by_filter_title=Ordenar por
+order_by_filter_relevance=Relevancia
+order_by_filter_latest=Reciente
+order_by_filter_az=A-Z
+order_by_filter_rating=Valoración
+order_by_filter_trending=Tendencia
+order_by_filter_views=Vistas
+order_by_filter_new=Nuevos
+genre_condition_filter_title=Condición de género
+genre_condition_filter_or=O
+genre_condition_filter_and=Y
+adult_content_filter_title=Contenido adulto
+adult_content_filter_all=Todos
+adult_content_filter_none=Ninguno
+adult_content_filter_only=Solo
+genre_filter_header=Es posible que el filtro de géneros no funcione correctamente
+genre_filter_title=Géneros
+genre_missing_warning=Presione 'Restablecer' para intentar mostrar los géneros
+alt_names_heading=Nombres Alternativos:
```

Comment:
make sure there is a space at the end, "alternatives: "

or change madara code to add space